app/views/admin/videos/new.html.haml

```diff
@@ -0,0 +1,21 @@
+%section.admin_add_video
+  .container
+    .row
+      .col-md-10.col-md-offset-1
+        = bootstrap_form_for [:admin, @video], html: { class: "form-horizontal" } do |f|
```

Comment:
You need to use `layout: :horizontal` now as stated in the gem docs. I assume your form doesn't look right currently.